benchmark/bench-thread.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+const { workerData: benchmark, parentPort } = require("worker_threads");
+
+const Benchmark = require("benchmark");
```

Comment:
We already have cronometro in the package. Can you change this with `cronometro`